src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -97,6 +99,46 @@ namespace PhotovoltaicThermalCollectors {
         }
     };
 
+    struct BIPVTModelStruct
+    {
+
+        // Members
+        std::string Name;
+        std::string OSCMName;         // OtherSideConditionsModel
+        int OSCMPtr;                  // OtherSideConditionsModel index
+        int SchedPtr;                 // Availablity schedule
+        Real64 PVEffGapWidth;         // Effective Gap Plenum Behind PV modules (m)
+        Real64 EffCollHeight;         // Effective Overall Width of Collector (m)
+        Real64 EffCollWidth;          // Effective Overall Height of Collector (m)
+        Real64 PVCellTransAbsProduct; // PV cell Transmittance-Absorptance prodiuct
+        Real64 BackMatTranAbsProduct; // Backing Material Normal Transmittance-Absorptance Product
+        Real64 CladTranAbsProduct;    // Cladding Normal Transmittance-Absorptance Product
+        Real64 PVAreaFract;           // Fraction of collector gross area covered by PV module
+        Real64 PVCellAreaFract;       // Fraction of PV cell area to module area
+        Real64 PVRTop;                // PV module top thermal resistance (m2-DegC/W)
+        Real64 PVRBot;                // PV module bottom thermal resistance (m2-DegC/W)
+        Real64 PVGEmiss;              // Emissivity PV modules
+        Real64 BackMatEmiss;          // Emissivity of backing material
+        Real64 ThGlass;               // Glass thickness (m)
+        Real64 RIndGlass;             // Glass refraction index
+        Real64 ECoffGlass;            // Glass extinction coefficient
+        Real64 LastCollectorTemp;     // store previous temperature (DegC)
+        Real64 CollectorTemp;         // average solar collector temp (DegC)
+        Real64 Tplen;                 // modeled drybulb temperature for air through BIPVT channel (DegC)
+        Real64 Tcoll;                 // modeled temperature of BIPVT channel surface on PV side (DegC)
+        Real64 HrPlen;                // Modeled radiation coef for OSCM (W/m2-C)
+        Real64 HcPlen;                // Modeled Convection coef for OSCM (W/m2-C)
+
+        // Default Constructor
```

Comment:
Initialize in declaration, remove default construct.

Reply:
updated